components/NursePage/NursePage.js

```diff
@@ -0,0 +1,32 @@
+import useSWR from "swr";
+import { useRouter } from "next/router";
+import Link from "next/link";
+import Image from "next/image";
+
+export default function NursePage() {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const { data, isLoading } = useSWR(`/api/nurses/${id}`);
+
+  if (isLoading) {
+    return <h1>Loading...</h1>;
+  }
+
+  if (!data) {
+    return <h1>Maybe he resigned without warning! 🤬</h1>;
+  }
```

Comment:
Those are not actually headings ^^ Consider using a more generic element (styled like you want!)